code/modules/shuttle/on_move.dm

```diff
@@ -13,6 +13,16 @@
         return 0
     . = ..()
 
+/obj/effect/landmark/onShuttleMove()
+    // Drop invisibility juuust enough to get moved
+    invisibility = 100
+    . = ..()
+    invisibility = 101
+
+/obj/effect/landmark/shuttle_import/onShuttleMove()
+    // Unless you're THIS landmark
+    return 0
+
```

Comment:
Not sure. TG does the same invisibility exclusion, and if it's only for that landmark, they would probably have an exclusion just for it.